src/main/kotlin/com/mituuz/fuzzier/FuzzyMover.kt

```diff
@@ -183,11 +183,7 @@ class FuzzyMover : FuzzyAction() {
             val projectFileIndex = ProjectFileIndex.getInstance(project)
             val projectBasePath = project.basePath
 
-            val contentIterator = if (!component.isDirSelector) {
-                projectBasePath?.let { stringEvaluator.getContentIterator(it, searchString, listModel) }
-            } else {
-                projectBasePath?.let { stringEvaluator.getDirIterator(it, searchString, listModel) }
```

Comment:
Yup, the `getDirIterator` is used for the file mover. Combining these won't work as expected because now you there are files present on the directory selection.